src/main/java/com/imaginarycode/minecraft/redisbungee/RedisBungeeListener.java

```diff
@@ -244,7 +245,9 @@ public void run() {
                             return;
                     }
 
-                    ((Server) event.getSender()).sendData("redisbungee:RedisBungee", out.toByteArray());
+                    final Server servSender = (Server) event.getSender();
+                    servSender.sendData("legacy:RedisBungee", out.toByteArray());
+                    servSender.sendData("RedisBungee", out.toByteArray());

```

Comment:
It might be better to send the response just to the channel we received the request from